x-pack/test/api_integration/apis/management/index_management/inference_endpoints.ts

```diff
@@ -54,7 +56,7 @@ export default function ({ getService }: FtrProviderContext) {
         inferenceEndpoints.some(
           (endpoint: InferenceAPIConfigResponse) => endpoint.inference_id === inferenceId
         )
-      ).to.eql(true, `${inferenceId} not found in the GET _inference/_all response`);
```

Comment:
nit: I wonder if it might be better to leave this change so that we don't forget to add it back when we unskip the tests. WDYT?